media/live-seekable-range.html

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Strange - on Linux, Google Chrome 60.0.3112.101 (Official Build) (64-bit), it looks like this API is supported. Maybe Blink's MediaSourceExperimental RuntimeEnabledFlag is enabled for Googlers.
Regardless, I observed eventual "Script error. (Your browser may not support this feature.)" -- Other than clicking a bunch of the buttons and waiting, and waiting some more, then switching tabs/windows and coming back, I'm not sure how to repro that error. Note that initially that same row in the log shows just "Warning! Keep in mind this sample has been tested with Chrome 62."